Makefile

```
@@ -10,7 +10,7 @@ BINARIES=$(shell cd $(ROOT_DIR)/cmd && ls -1 | grep -v ^common)
 # Common flags for fuzz tests
 FUZZ_FLAGS ?= -run=^$ -fuzztime=10s
 
-.PHONY: mod-tidy test test-match bench fuzz format clean play play-fmt play-flat build download-plutus-tests
+.PHONY: mod-tidy test test-match bench fuzz format clean download-plutus-tests
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)